### DISCUSSION TITLE
[@types/wordpress__components] ToggleControl missing `disabled` prop definition

### DISCUSSION THREAD

**Discussion by ivan-ottinger** [DefinitelyTyped/DefinitelyTyped#63582] [Issues with a @types package]
[ToggleControl type definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__components/toggle-control/index.d.ts#L5) is missing `disabled` prop.

We noticed the issue while working on https://github.com/Automattic/wp-calypso/pull/70981.

The following change will likely fix the issue:

```diff
@@ -14,6 +14,8 @@
          * A function that receives the checked state as input.
          */
         onChange?(isChecked: boolean): void;
+
+        disabled?: boolean | undefined;
     }
 }
 declare const ToggleControl: ComponentType<ToggleControl.Props>;
```

Thanks to @sirbrillig for pointing me to the right direction.

[a comment by an automated account is not shown]

**Comment by manzoorwanijk** [comment]
Yes, I can see it's documented [here](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/toggle-control#disabled).

A PR to update the type would be great.

**Comment by ivan-ottinger** [comment]
Thanks @manzoorwanijk, I will prepare the PR.

**Comment by ivan-ottinger** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63596 is now ready for review.
